src/fetch/PlaywrightFetcher.js

```diff
@@ -167,6 +167,22 @@ export const PlaywrightFetcher = class extends BaseFetcher {
     }
   }
 
+  async select(ctx, selector, options) {
```

Comment:
I think we can remove this.

On `dev` it now does each action that it learns, and takes `ok` based on if that action worked in Playwright. This implicitly checks if we're able to select it. It doesn't ask the AI if the action was successful, but it's a reasonable approach for now.